src/Microsoft.IdentityModel.Clients.ActiveDirectory/Features/WinCommon/AuthenticationContext.cs

```diff
@@ -36,314 +36,273 @@ namespace Microsoft.IdentityModel.Clients.ActiveDirectory
     /// <summary>
     /// Extension class to support confidential client flows.
```

Comment:
The comment needs updating. It's no longer an extension class (but a partial implementation) to support confidential client flows

Reply:
fixed